src/plugins/data_view_management/public/components/edit_index_pattern/edit_index_pattern.tsx

```diff
@@ -161,12 +119,33 @@ export const EditIndexPattern = withRouter(
     const docsUrl = kibana.services.docLinks!.links.elasticsearch.mapping;
     const userEditPermission = dataViews.getCanSaveSync();
 
+    const warning =
+      indexPattern.namespaces && indexPattern.namespaces.length > 1 ? (
+        <FormattedMessage
+          id="indexPatternManagement.editDataView.deleteWarningWithNamespaces"
+          defaultMessage="The data view {dataViewName} will be deleted from every space it is shared in. You can't undo this action."
+          values={{
+            dataViewName: <EuiCode>{indexPattern.title}</EuiCode>,
+          }}
+        />
+      ) : (
+        <FormattedMessage
+          id="indexPatternManagement.editDataView.deleteWarning"
+          defaultMessage="The data view {dataViewName} will be deleted. You can't undo this action."
```

Comment:
Another option is to put the name at the end of the sentence:

Delete the data view {dataViewName}.  You can't undo this action.

Reply:
It also removes the passive "will be deleted".